packages/cfd/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "compilerOptions": {
         "paths": {
             "_common/*": ["./src/_common/*"],
+            "Assets/*": ["./src/Assets/*"],
```

Comment:
I guess this is not valid. we don't have assets directory in cfd :thinking:

Reply:
wrong tsconfig file 😓 🙏 will edit it